lib/browserstack/local.rb

```diff
@@ -101,8 +101,8 @@ def start(options = {})
     end
   end
 
-  def isRunning
-    return true if (!@pid.nil? && Process.kill(0, @pid)) rescue false
+  def is_running?
```

Comment:
For backward compatibility, can you also keep `isRunning` defined, but it should just call new method `is_running?`. Don't want users upgrading and having broken pipelines.

Also please update Readme with new method